adapters/grid/grid.go

```diff
@@ -427,6 +432,9 @@ func Builder(bidderName openrtb_ext.BidderName, config config.Adapter, server co
 }
 
 func getBidMeta(ext json.RawMessage) (*openrtb_ext.ExtBidPrebidMeta, error) {
+	if ext == nil {
+		return nil, errors.New("nil json.RawMessage passed to getBidMeta")
```

Comment:
What do you think about changing the error message to `"nil ext passed to getBidMeta"`

Reply:
sure - changed in https://github.com/prebid/prebid-server/commit/6977017fecf71105977c6ca7bb75cc0fac731f54